qa/suites/fs/fscrypt/tasks/0-client.yaml

```diff
@@ -1,7 +0,0 @@
-teuthology:
-  postmerge:
-    # Once can we make sure the distro kernels have included the fscrypt feature
-    # or the ceph-fuse have supported the fscrypt feature we can remove this
-    # restriction.
-    - if not is_kupstream() then reject() end
-task:
```

Comment:
Obviously it's thrilling that we can now remove this but we also need a test added that shows a kclient can create an encrypted directory tree and then a ceph-fuse client can read it. Then vice versa.

Reply:
Yes you are correct.
`tasks.cephfs.test_fscrypt.TestFSCryptRMW` has tests with multiple mounts/clients. These tests currently only support the same type of client (fuse and fuse or kclient and kclient). Need to work out how to test multiple types in same test.

Reply:
This is completed, checkout commit: https://github.com/ceph/ceph/pull/61137/commits/e1e2d480fa47f4de2fb96dc25e7330351dfa9a97